packages/core/src/classes.ts

```diff
@@ -41,12 +41,11 @@ export interface UXObject extends Obj {
   icon?: Asset
 }
 
-export interface Type<T extends PropertyType> extends Emb, UXObject {
+export interface Type<T extends PropertyType> extends Emb, UXObject { // eslint-disable-line
 }
```

Comment:
Please specify disabled rule

Reply:
OK, by "please specify disabled rule" I mean please add parameter describing the rule, such as `// eslint-disable-line no-void`. It's good to have in future if we change eslint rule settings, so we can remove relevant comments.

Reply:
Done.